### PR TITLE
Update docs to Quarkus

### PR DESCRIPTION
**Pull request by yurloc**
### Referenced pull requests

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc [hunk 1]
```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
 
 [source,shell]
 ----
-mvn spring-boot:run
+mvn compile quarkus:run
```
Comment (Christopher-Chianelli) [inline optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:72]: There is no "run" goal in the Quarkus plugin; use `mvn quarkus:dev` for this.

**Review by Christopher-Chianelli — CHANGES_REQUESTED** [review]
Beside the `quarkus:dev`; LGTM

**Review by Christopher-Chianelli — APPROVED** [review]

File: optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc [hunk 2]
```diff
@@ -53,8 +53,8 @@ Finally, the back end module implements additional supporting functionality, suc
 In the next sections you will learn how to configure and run the back end in development mode.
 To learn more about the back end code architecture, see <<appendix-backend-architecture#backend-architecture>>.
 
-[[run-spring-boot-maven-plugin]]
-=== Running the back end using Spring Boot Maven plugin
+[[run-quarkus-maven-plugin]]
+=== Running the back end using Quarkus Maven plugin
```
Comment (emmurphy1) [inline optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:57]: the Quarkus (I seem to have missed it for Spring Boot as well :-))

File: optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc [hunk 3]
```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
...
 ----
 
-=== Automatic restart
+=== Quarkus development mode
 
-https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-devtools-restart[Automatic restart]
-is provided by Spring Boot DevTools and only works when the back end is running using <<run-spring-boot-maven-plugin, Spring Boot Maven Plugin>>.
-It scans files on the classpath, so you only need to recompile your changes to trigger application restart.
-No IDE configuration is needed.
+In development mode, the backend automatically restarts whenever you refresh the browser tab where the front end runs if there are any changes in the back-end source code or configuration.
```
Comment (emmurphy1) [inline optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:77]: change where to and:
and the front end runs...

File: optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc [hunk 4]
```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
...
 
-If your IDE has a compile-on-save feature (for example Eclipse or NetBeans), you just need to save the files that have changed since the last compilation.
+Learn more about https://quarkus.io/guides/maven-tooling#development-mode[Quarkus development mode].
 
-IntelliJ IDEA saves changes automatically and you need to select either menu:Build[Recompile], which recompiles the file in the active tab, or menu:Build[Build Project] which recompiles all changes.
-See https://www.jetbrains.com/help/idea/compiling-applications.html[Compile and build applications with IntelliJ IDEA].
+=== Running the back end from IntelliJ IDEA Ultimate
 
-=== Running the back end from IntelliJ IDEA
+IntelliJ IDEA Ultimate has a bundled Quarkus plugin that automatically creates run configurations for modules using the Quarkus framework.
```
Comment (emmurphy1) [inline optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:83]: plug-in, not plugin, in all cases. Maybe search for you in your files? (IBM Style Guide)

File: optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc [hunk 5]
```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
...
 
-You can learn more about configuring a Spring Boot application on the
-https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config[Spring Boot Externalized Configuration] page.
+To override the default configuration temporarily, use system properties (`-Dproperty=value`).
 
-[TIP]
-Use `src/main/resources/application-local.properties` to store your personal configuration without affecting the Git working tree.
+==== The `.env` file
+
+To override the default configuration permanently, for example to store configuration that is specific for your development environment, use the `optaweb-vehicle-routing-backend/.env` file.
```
Comment (emmurphy1) [inline optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:103]: to store a configuration...
specific to your environment...

File: optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc [hunk 6]
```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
...
-See the
-https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-custom-log-levels[Spring Boot Logging]
-documentation for more information.
-
-Following are examples of properties you can use to control logging level of some parts of the application:
-
-- Use `logging.level.org.optaweb.vehiclerouting=debug` to enable debug level for the back end code.
-- Use `logging.level.org.optaplanner.core=warn` to reduce OptaPlanner logging.
-- Use `logging.level.org.springframework.web.socket=trace` to access more details when investigating problems with WebSocket connection.
+See https://quarkus.io/guides/logging[Quarkus - Configuring Logging] for more information.
```
Comment (emmurphy1) [inline optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:115]: Not major, but we generally use this construction:
For more information , see....

File: optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc (hunk 3 above)
Comment (yurloc) [reply optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:77]: The suggested change would change the meaning of the sentence. The meaning is:

The back end restarts whenever 1) you refresh the browser tab where the front end runs [and] 2) if there are any changes...

Comment (yurloc) [reply optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc:77]: Feel free to propose another change that would convey the intended meaning in a more simple and clearer way.

**Pull request merged**